kong/cmd/utils/prefix_handler.lua

```diff
@@ -378,6 +378,54 @@ local function write_process_secrets_file(path, data)
   return true
 end
 
+local function pre_create_lmdb(conf)
+  local prefix = conf.prefix
+  local lmdb_path = conf.lmdb_environment_path or "dbless.lmdb"
+  local user = string.match(conf.nginx_user or "", "%w+") or "`whoami`"
```

Comment:
This is probably not right. It will make the directory owned by `root`, which is not what we want.

Why is `conf.nginx_user` empty in the first place, we have default value set for these.

Reply:
Should we give it a default value `"kong"`?

Reply:
I am wondering why did we not see the default value from `kong.configuration` in the first place.

Reply:
I checked the code of `conf_loader`, it is set to `nil`.

https://github.com/Kong/kong/blob/cb5e0460709ea7d115029ec3711158cdcd908c7a/kong/conf_loader/init.lua#L1579